### DISCUSSION TITLE
Is it possible to change the summary label on MultiSelectDropDown?

### DISCUSSION THREAD

**Discussion by stenci** [anvilistas/anvil-extras#219] [Ideas]
It looks like the MultiSelectDropDown shows the text of the selected items if there are 3 or fewer selected items, otherwise shows the number of selected items, something like "4 items selected".

I would like to change that logic to either always show the full list if selected items or show a custom string.

Looking at the code I was expecting to see something like `dropdown_component.text = summary_text`, instead I got... nothing. No components, just some HTML and js and ts.

So... I don't really know what I want, I would need to play with it a little. Perhaps a `max_number_of_items_in_summary_text` or a `max_summary_text_length` property that will affect when to switch from listing the items to showing a summary label would work. Or perhaps an event that allows me to explicitly set the summary text in my own function.

But looking at that code I couldn't figure out where the summary text is generated.

Any help?

**Comment by s-cork** [comment]
This is set in 
a property within the html. The property that has "count > 3". 

The JavaScript library has an api and I'm using that to set an option. 

Here's the library and their api for that property https://developer.snapappointments.com/bootstrap-select/examples/#selected-text-format. 


If we decide the best python api we can then work out how to implement it. The easiest is just to expose the js library's api.

**Comment by s-cork** [comment]
@stenci - did you want to work on this?
